distributed/scheduler.py

```diff
@@ -4997,9 +5000,11 @@ async def get_profile_metadata(
         else:
             workers = set(self.workers) & set(workers)
         results = await asyncio.gather(
-            *(self.rpc(w).profile_metadata(start=start, stop=stop) for w in workers)
+            *(self.rpc(w).profile_metadata(start=start, stop=stop) for w in workers),
+            return_exceptions=True
         )
 
+        results = [r for r in results if not isinstance(r, Exception)]
```

Comment:
Are there ever circumstances where everything fails and we should report it?  "No" is a fine answer, I'm just bringing this up in case we can think of something.

Reply:
Well, if we assume that any sort of code changes will be caught by code and we restrict ourselves to runtime failures I tend to say no (apart from systematic failure of network or similar catastrophic events). I wouldn't be surprised to be proved wrong, though. Shall we add at least some kind of logging/warning?